src/renderer.rs

```diff
@@ -10,9 +10,17 @@ use regex::Regex;
 use crate::preprocessor::GraphvizBlock;
 use tokio::io::AsyncWriteExt;
 
+#[derive(Default)]
+pub struct GraphvizRendererConfig {
+    pub link_to_file: bool,
+}
+
 #[async_trait]
 pub trait GraphvizRenderer {
-    async fn render_graphviz<'a>(block: GraphvizBlock) -> Result<Vec<Event<'a>>>;
+    async fn render_graphviz<'a>(
+        block: GraphvizBlock,
+        _config: &GraphvizRendererConfig,
```

Comment:
nit: I don't believe we need the `_` for config here

Reply:
Ah, forgot to make this change. I'll have to get it next time.